Doc/library/turtle.rst

```diff
@@ -19,9 +19,27 @@
 Introduction
 ============
 
-Turtle graphics is a popular way for introducing programming to kids.  It was
-part of the original Logo programming language developed by Wally Feurzeig,
-Seymour Papert and Cynthia Solomon in 1967.
+Turtle graphics is an implementation of `the popular geometric drawing tools
+introduced in Logo <https://en.wikipedia.org/wiki/Turtle_
+(robot)>`_, developed by Wally Feurzeig, Seymour Papert and Cynthia Solomon
+in 1967.
+
+In Python, turtle graphics provides a representation of a physical "turtle"
+(a little robot with a pen) that draws on a sheet of paper on the floor.
+
+It's a very effective and well-proven way for learners to encounter
```

Comment:
Feel free to ignore :)

> ‘Substitute “damn” every time you’re inclined to write “very”. Your editor will delete it and the writing will be just as it should be.’ William Allen White

https://www.theguardian.com/guardian-observer-style-guide-w



```suggestion
It's an effective and well-proven way for learners to encounter
```